enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/grpc/ExtAuthService.java

```diff
@@ -197,6 +228,12 @@ private CheckResponse buildResponse(CheckRequest request, ResponseObject respons
         }
     }
 
+    private String getDirectResponseSoapBody() {
```

Comment:
can remove this

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/5e1b89d27c7112fe450be54d1bbeaf4f262a6ef7